warning: .../dog-0.1.0/Cargo.toml: `default_features` is deprecated in favor of `default-features` and will not work in the 2024 edition
(in the `datetime` dependency)
warning: .../dog-0.1.0/Cargo.toml: `default_features` is deprecated in favor of `default-features` and will not work in the 2024 edition
(in the `regex` dependency)

---

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,7 +40,7 @@ atty = "0.2"
 getopts = "0.2"
 
 # transaction ID generation
-rand = "0.7"
+rand = "0.8"
 
 # json
 serde = "1.0"
@@ -54,8 +54,8 @@ log = "0.4"
 ipconfig = { version = "0.2" }
 
 [build-dependencies]
-datetime = { version = "0.5.1", default_features = false }
-regex = { version = "1.3", default_features = false, features = ["std"] }
+datetime = { version = "0.5.1", default-features = false }
+regex = { version = "1.3", default-features = false, features = ["std"] }
 
 [dev-dependencies]
 pretty_assertions = "0.6"
@@ -64,3 +64,7 @@ pretty_assertions = "0.6"
 default = ["tls", "https"]
 tls = ["dns-transport/tls"]
 https = ["dns-transport/https"]
+
+[patch.crates-io]
+# all: no SYS_getrandom
+getrandom = { path = "../getrandom-0.2.16" }
